relay-event-normalization/src/replay.rs

```diff
@@ -51,9 +51,13 @@ pub fn validate(replay: &Replay) -> Result<(), ReplayError> {
         .value()
         .ok_or_else(|| ReplayError::InvalidPayload("missing segment_id".to_string()))?;
 
-    if segment_id > u16::MAX as u64 {
+    // Each segment is expected to be 5 seconds in length. We take the number of seconds in
```

Comment:
I think it is possible for segments to be over 5 seconds fwiw.

Reply:
That's okay with me.  I'm hoping to limit abuse in a reasonable way.  Do you have a different number in mind?  More buffer, less?

Reply:
I'm good w/ this, just wanted to point it out